src/main/java/seedu/address/commons/util/JsonUtil.java

```diff
@@ -49,14 +49,16 @@ static <T> T deserializeObjectFromJsonFile(Path jsonFile, Class<T> classOfObject
     }
 
     /**
-     * Returns the Json object from the given file or {@code Optional.empty()} object if the file is not found.
-     * If any values are missing from the file, default values will be used, as long as the file is a valid json file.
+     * Returns the Json object from the given file or {@code Optional.empty()} object if the file is
```

Comment:
Not strictly related to this PR but as long as we are touching these lines, perhaps use `JSON` or `json` instead of `Json`?